### DISCUSSION TITLE
Flask deploy error

### DISCUSSION THREAD

**Discussion by realz99th** [vercel/community#3857] [Bug Report]
### Page to Investigate

check module

### Steps to Reproduce

Build Failed
Command failed with exit code 1: du -ms /vercel/path0/node_modules du: cannot access ‘/vercel/path0/node_modules’: No such file or directory

{
    "version": 2,
    "builds": [
        {
            "src": "./index.py",
            "use": "@vercel/python"
        }
    ],
    "routes": [
        {
            "src": "/(.*)",
            "dest": "/"
        }
    ]
}

**Comment by binarykore** [comment]
Same here..

**Comment by raiperez** [comment]
Having the same issue

**Comment by julkaar9** [comment]
Same with golang, 
Error logs
```
Build Failed
Command failed with exit code 1: du -ms /vercel/path1/node_modules du: cannot access ‘/vercel/path1/node_modules’: No such file or directory
```
Build log stops at `Deploying outputs...`

**Comment by binarykore** [comment]
It's fixed now.. Successfully redeployed mine.. :)

**Comment by majidemad** [comment]
was this a global bug in the platform?

**Comment by amyegan** [comment]
I'm glad to see it's working now. Please let know if it happens to you again 🙏
